api/db/seeds/data/team-evaluation/autonomous-courses/create-autonomous-courses.js

```diff
@@ -0,0 +1,66 @@
+import dayjs from 'dayjs';
+import { AUTONOMOUS_COURSES_ID } from '../constants.js';
+import { PRO_ORGANIZATION_USER_ID, ALL_ORGANIZATION_USER_ID } from '../../common/constants.js';
+import { REAL_PIX_SUPER_ADMIN_ID } from '../../common/common-builder.js';
+import * as tooling from '../../common/tooling/index.js';
+
+export default async function initUser(databaseBuilder) {
+  const superAdminId = REAL_PIX_SUPER_ADMIN_ID;
+
+  // 1. Create autonomous-courses specific organization
+  const { id: specificOrganizationId } = await databaseBuilder.factory.buildOrganization({
+    id: process.env.AUTONOMOUS_COURSES_ORGANIZATION_ID || AUTONOMOUS_COURSES_ID,
+    type: 'PRO',
+    externalId: 'AUTONOMOUS_COURSES',
+    name: 'Autonomous courses organizaition',
+    isManagingStudents: false,
+    adminIds: [PRO_ORGANIZATION_USER_ID],
+    memberIds: [ALL_ORGANIZATION_USER_ID],
+  });
+
+  // 2. Create membership between SCO user and our specific organization
+  databaseBuilder.factory.buildMembership({
+    userId: superAdminId,
+    organizationId: specificOrganizationId,
+    organizationRole: 'ADMIN',
+  });
+
+  // 3. Create autonomous-courses (public and simplified-access campaigns linked to our specific organization)
+  for (let i = 1; i < 6; i++) {
+    // 3.1. Create some target-profiles owned by our specific organization
+    await tooling.targetProfile.createTargetProfile({
+      databaseBuilder,
+      ownerOrganizationId: specificOrganizationId,
+      targetProfileId: AUTONOMOUS_COURSES_ID + i,
+      isPublic: true,
+      isSimplifiedAccess: true,
+      name: `Profil-cible pour parcours autonome n°${i}`,
+      description: 'Profil cible pour parcours autonome',
+      category: ['Les 16 compétences', 'Thématiques', 'Parcours sur-mesure', 'Parcours prédéfinis', 'Autres'][i - 1],
```

Comment:
Faire le for à partir du tableau des catégories

Reply:
Au final, on ne va pas le faire parce qu'on ne peut pas utiliser de callback asynchrone dans un `forEach`